Graphics/MusicWheelItem Song NormalPart/get_lamp.lua

```diff
@@ -0,0 +1,86 @@
+local args = {...}
+local graphic = args[1]
+local pn = args[2]
+
+LampColors = {
+    [0]={1,1,1,0},
+    color "#ffffff",
```

Comment:
The colors should be taken from SL.JudgmentColors from the respective game mode so we don't need this table.
https://github.com/Simply-Love/Simply-Love-SM5/blob/release/Scripts/SL_Init.lua#L190-L205

Reply:
Done, though the colors don't pop a whole lot, maybe I should add some sort of glow to it?

Reply:
I think the best thing would be to pulse between too related colors (with some outer glow I think might look nice). Try out whatever you think looks nice!